pyproject.toml

```diff
@@ -25,17 +25,18 @@ classifiers = [
 ]
 dependencies = ["mne"]
 dynamic = ["version"]
+readme = {content-type = "text/x-rst", file = "README.rst"}
 
 [project.urls]
 "Bug Tracker" = "https://github.com/wmvanvliet/mne-faster/issues/"
 "Source Code" = "https://github.com/wmvanvliet/mne-faster"
+"Homepage" = "https://github.com/wmvanvliet/mne-faster"
 
 [tool.setuptools]
 packages = ["mne_faster"]
 
 [tool.setuptools.dynamic]
 version = {attr = "mne_faster.__version__"}
-readme = {file = ["README.rst"]}
```

Comment:
I think this was in the wrong place, moving it should fix the description issue :crossed_fingers: